lib/std/mem.zig

```diff
@@ -876,6 +874,63 @@ pub fn indexOfPos(comptime T: type, haystack: []const T, start_index: usize, nee
     return null;
 }
 
+fn boyerMooreHorspoolPreprocess(pattern: []const u8, table: []usize) void {
```

Comment:
Shouldn't `table` be `*[256]usize` (or `*[math.maxInt(u8) + 1]usize`)?

Reply:
fixed